etna/models/nn/tft.py

```diff
@@ -181,7 +204,49 @@ def forecast(self, ts: TSDataset) -> TSDataset:
 
         predicts = self.model.predict(prediction_dataloader).numpy()  # type: ignore
         # shape (segments, encoder_length)
-
         ts.loc[:, pd.IndexSlice[:, "target"]] = predicts.T[-len(ts.df) :]
+
+        if prediction_interval:
+            quantiles_predicts = self.model.predict(  # type: ignore
```

Comment:
You are right: fitting is working but during prediction here quantiles aren't generated: only one value is returned for each point instead of `len(quantiles)` values.

Reply:
It looks like we should print Warning.